test/TritonGPU/accelerate-matmul.mlir

```diff
@@ -74,6 +75,86 @@ module attributes {"triton_gpu.compute-capability" = 80 : i32, "triton_gpu.num-c
 
 // -----
 
+// CHECK-89: #[[$MMA:.+]] = #triton_gpu.nvidia_mma<{versionMajor = 2, versionMinor = 0, warpsPerCTA = [4, 2], instrShape = [16, 8]}>
```

Comment:
those test seem a bit redundant, what different scenario are you trying to catch? Probably a single test is enough. Also the CHECK (`CHECK-89: tt.dot {{.*}} -> tensor<64x64xf32, #[[$MMA]]>`)  doesn't seem to verify that the operands are float8 types so I don't think it validates the changes? Wouldn't those test pass without your changes?

Reply:
You are right. I have updated it, please take a look again